Desktop/html/index-jasp.html

```diff
@@ -13,6 +13,10 @@
 	<link rel="stylesheet" href="css/lightTheme-jasp.css"					type="text/css" id="style"/>
 	<link rel="stylesheet" href="css/printing.css"							type="text/css" />
 
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/styles/default.min.css">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/highlight.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/languages/r.min.js"></script>
```

Comment:
yes I know that, I just left it for now to live debugging, once I'm done debugging I'll import from local:-) Because reloading resources from qtwebengine by `qrc://` during live debug phase requires compiling every time.